test/upgrade-e2e/post_upgrade_test.go

```diff
@@ -86,7 +86,7 @@ func TestClusterExtensionAfterOLMUpgrade(t *testing.T) {
 		assert.Equal(ct, metav1.ConditionTrue, cond.Status)
 		assert.Equal(ct, ocv1alpha1.ReasonSuccess, cond.Reason)
 		assert.Contains(ct, cond.Message, "Installed bundle")
-		if assert.NotEmpty(ct, clusterExtension.Status.Install.Bundle) {
```

Comment:
`clusterExtension.Status.Install.Bundle` is not a pointer and it doesn't make much sense to check for it being empty: we can just check the value (`Version`) we care about after checking that `clusterExtension.Status.Install` is not `nil`.